src/main/java/com/akademia/educators_back/service/ProblemService.java

```diff
@@ -0,0 +1,26 @@
+package com.akademia.educators_back.service;
+
+import com.akademia.educators_back.entity.ProblemEntity;
+import com.akademia.educators_back.repository.ProblemRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ProblemService {
+
+    ProblemRepository problemRepository;
+
+    public void addProblemToDB(ProblemEntity problem) {
+        problemRepository.save(problem);
+    }
+
+    public void deleteProblemFromDB(ProblemEntity problem) {
+        problemRepository.delete(problem);
+    }
+
+    public void updateProblem(Long id, String newTitle, String newQuestion) {
```

Comment:
Jeśli metoda ma więcej niż 2 paramtery to już zwraca uwagę. W tym wypadku wystarczyłoby gdybyśmy dostawali wszystkie zmiany w postaci obiektu DTO. Jeśli nie mamy jeszcze tej warstwy to lepiej byłoby wydzielić dwie osobne metody: jedna na update tytułu druga na updade pytania.